### PR TITLE
Remove Add Ask

### LINKED ISSUE #1794: Remove use of `add-ask` command, and remove command
### Description

The `add-ask` command has been deprecated. Also the command does not properly configure the `miner.storagePrice`, so using it is not valid. We should remove all use of `add-ask` in our tests and deployments.

Related issue for infra: https://github.com/filecoin-project/go-filecoin-infra/issues/150

### Acceptance criteria

- `add-ask` is no longer used in functional retrieval test
- `add-ask` is removed from FAT lib.
- Remove `add-ask` from cli.

### Risks + pitfalls

Removing the use of `add-ask` is a bit of work. We could also update it to set the price as well.

### Where to begin

https://github.com/filecoin-project/go-filecoin/blob/master/functional-tests/retrieval
https://github.com/filecoin-project/go-filecoin/blob/master/functional-tests/devnet-retrieval
https://github.com/filecoin-project/go-filecoin/blob/master/tools/fat/action_miner.go
https://github.com/filecoin-project/go-filecoin/blob/master/commands/miner.go

### PR DESCRIPTION
**Pull request by rkowalick**
Resolves #1794 

Additionally, remove the unnecessary `fork/join` stuff in the retrieval shell scripts

### CODE REVIEW

File: functional-tests/devnet-retrieval [hunk 1]
```diff
@@ -125,19 +123,13 @@ echo "miner node starts mining..."
 ./go-filecoin mining start \
   --repodir="$MN_REPO_DIR"
 
-echo ""
```
Comment (phritz) [inline functional-tests/devnet-retrieval]: keep an eye on nightly

File: functional-tests/devnet-retrieval [hunk 2]
```diff
@@ -125,19 +123,13 @@ echo "miner node starts mining..."
...
 echo ""
-echo "block until miner peer id-update message appears in chains..."
-join 104
-join 105
-join 106
-join 107
+echo "waiting until messages are in blockchain..."
+./go-filecoin message wait "${MINER_UPDATE_PID_MSG_CID}" --repodir="${MN_REPO_DIR}"
+./go-filecoin message wait "${MINER_UPDATE_PID_MSG_CID}" --repodir="${CL_REPO_DIR}"
+./go-filecoin message wait "${MINER_ADD_ASK_MSG_CID}" --repodir="${CL_REPO_DIR}"
```
Comment (phritz) [inline functional-tests/devnet-retrieval]: maybe note the reason we dont have to wait on the miner node for the add ask, presumably bc there is a wait in set price above

File: functional-tests/devnet-retrieval (hunk 1 above)
Comment (rkowalick) [reply functional-tests/devnet-retrieval]: I think this script is going to be deleted. No one is running it and its role will be likely be replaced by something FAST-based.

File: functional-tests/devnet-retrieval (hunk 2 above)
Comment (rkowalick) [reply functional-tests/devnet-retrieval]: Done

**Pull request merged**